### PR TITLE
Add scheme for organizationId and websiteId

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request adds the following:
- Server descriptions for reuse.
- An organizationId scheme.
- A websiteId scheme.

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml [hunk 1]
```diff
@@ -34,9 +34,7 @@ allOf:
         readOnly: true
       organizationId:
         readOnly: true
-        description: Organization ID.
-        allOf:
-          - $ref: ../../ResourceId.yaml
+        $ref: ../../../schemas/OrganizationId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml:37]: That doesn't seem correct, you have to combine fields via `allOf`. Please check preview, organizationId became writable https://preview.redoc.ly/rebilly-core/update-id-schemes/tag/Orders/operation/PostSubscription#!path=0/organizationId&t=request. Please re-check and fix all places.

File: openapi/components/schemas/Applications/OwnerApplicationInstance.yaml [hunk 2]
```diff
@@ -2,12 +2,7 @@ allOf:
   - $ref: ./ApplicationInstance.yaml
   - properties:
       organizationId:
-        description: |-
-          Unique organization identifier.
-          An organization is an entity that represents a company.
-          For more information, see [Obtain an organization ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
-        allOf:
-          - $ref: ../ResourceId.yaml
+        $ref: ../../schemas/OrganizationId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Applications/OwnerApplicationInstance.yaml:5]: Please use allOf

File: openapi/components/schemas/Common/CommonOrder.yaml [hunk 3]
```diff
@@ -32,11 +32,9 @@ properties:
       - partially-refunded
       - partially-paid
   websiteId:
-    description: The website identifier string.
+    $ref: ../WebsiteId.yaml
     x-sortable: true
     x-basic: true
-    allOf:
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonOrder.yaml:38]: Please use allOf

File: openapi/components/schemas/Common/CommonOrderPreview.yaml [hunk 4]
```diff
@@ -4,10 +4,8 @@ required:
   - items
 properties:
   websiteId:
-    description: The website identifier string.
+    $ref: ../WebsiteId.yaml
     writeOnly: true
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonOrderPreview.yaml:8]: please use allOf

File: openapi/components/schemas/Common/CommonTransaction.yaml [hunk 5]
```diff
@@ -6,14 +6,8 @@ properties:
     allOf:
       - $ref: ../ResourceId.yaml
   websiteId:
-    description: |-
-      Unique website identifier string. A website is where an
-      organization obtains a customer through a subscription. For more
-      information, see
-      [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
+    $ref: ../WebsiteId.yaml
     readOnly: true
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonTransaction.yaml:10]: Please use allOf

File: openapi/components/schemas/Customer.yaml [hunk 6]
```diff
@@ -25,13 +25,7 @@ properties:
     readOnly: true
   websiteId:
     x-sortable: true
```
Comment (ashkarpetin) [inline openapi/components/schemas/Customer.yaml:27]: Please use allOf

File: openapi/components/schemas/Websites/Website.yaml [hunk 7]
```diff
@@ -40,12 +40,7 @@ properties:
   organizationId:
     deprecated: true
     readOnly: true
-    description: |-
-      Unique organization identifier.
-      An organization is an entity that represents a company.
-      For more information, see [Obtain an organization ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
-    allOf:
-      - $ref: ../ResourceId.yaml
+    $ref: ../../schemas/OrganizationId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Websites/Website.yaml:43]: Please use allOf

File: openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml (hunk 1 above)
Comment (noon-dawg) [reply openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml]: Thanks for spotting this @ashkarpetin 🙏

File: openapi/components/schemas/GatewayAccount.yaml [hunk 8]
```diff
@@ -358,12 +358,8 @@ properties:
   organizationId:
     deprecated: true
     readOnly: true
-    description: |-
-      Unique organization identifier string.
-      An organization is an entity that represents company.
-      For more information, see [Obtain an organization ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
     allOf:
-      - $ref: ./ResourceId.yaml
+      -  $ref: ./Organizations/OrganizationId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccount.yaml:362]:
```suggestion
      - $ref: ./Organizations/OrganizationId.yaml
```

File: openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml [hunk 9]
```diff
@@ -34,9 +34,7 @@ allOf:
         readOnly: true
       organizationId:
         readOnly: true
-        description: Organization ID.
-        allOf:
-          - $ref: ../../ResourceId.yaml
+        $ref: ../../../schemas/Organizations/OrganizationId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml:37]: please use allOff

File: openapi/components/schemas/Websites/Website.yaml [hunk 10]
```diff
@@ -6,14 +6,8 @@ required:
   - serviceEmail
 properties:
   id:
-    description: |-
-      Unique website identifier string. A website is where an
-      organization obtains a customer through a subscription. For more
-      information, see
-      [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
+    $ref: ../Websites/WebsiteId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Websites/Website.yaml:9]: please use allOff

File: openapi/components/schemas/Websites/Website.yaml [hunk 11]
```diff
@@ -40,12 +34,8 @@ properties:
   organizationId:
     deprecated: true
     readOnly: true
-    description: |-
-      Unique organization identifier.
-      An organization is an entity that represents a company.
-      For more information, see [Obtain an organization ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
     allOf:
-      - $ref: ../ResourceId.yaml
+      -  $ref: ../../schemas/Organizations/OrganizationId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Websites/Website.yaml:38]:
```suggestion
      - $ref: ../../schemas/Organizations/OrganizationId.yaml
```

File: openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml [hunk 12]
```diff
@@ -34,9 +34,8 @@ allOf:
         readOnly: true
       organizationId:
         readOnly: true
-        description: Organization ID.
         allOf:
-          - $ref: ../../ResourceId.yaml
+          -  $ref: ../../../schemas/Organizations/OrganizationId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml:38]:
```suggestion
          - $ref: ../../../schemas/Organizations/OrganizationId.yaml
```

File: openapi/components/schemas/Websites/Website.yaml [hunk 13]
```diff
@@ -6,14 +6,9 @@ required:
...
   id:
-    description: |-
-      Unique website identifier string. A website is where an
-      organization obtains a customer through a subscription. For more
-      information, see
-      [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
     readOnly: true
     allOf:
-      - $ref: ../ResourceId.yaml
+      -  $ref: ../Websites/WebsiteId.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Websites/Website.yaml:11]:
```suggestion
      - $ref: ../Websites/WebsiteId.yaml
```

**Review by ashkarpetin — APPROVED** [review]

**Review by Boydbueno — APPROVED** [review]

**Pull request merged**